WorkloadManagementSystem/Agent/StalledJobAgent.py

```diff
@@ -443,10 +443,10 @@ def __checkLoggingInfo( self, jobID, jobDict ):
       if not startTime or startTime == 'None':
         startTime = jobDict['SubmissionTime']
 
-    if type( startTime ) in types.StringTypes:
+    if isinstance(startTime, basestring):
       startTime = fromString( startTime )
-      if startTime == None:
-        self.log.error( 'Wrong timestamp in DB', items[3] )
+      if startTime is None:
+        self.log.error( 'Wrong timestamp in DB' )
```

Comment:
The problem is that `items[3]1 is not necessary defined, as it is done inside a loop.